packages/react-app/src/components/ChallengeExpandedCard.jsx

```diff
@@ -133,92 +139,162 @@ const ChallengeExpandedCard = ({
-                  <span role="img" aria-label="lock icon">
-                    🔒
-                  </span>
-                  <chakra.span ml={1}>Locked</chakra.span>
-                </>
-              )}
-            </Button>
-            {!builderHasCompletedDependenciesChallenges && (
-              <Tooltip label={lockReasonToolTip}>
-                <IconButton icon={<QuestionOutlineIcon />} />
-              </Tooltip>
+    <Box borderBottom="2px" borderColor={borderColor} backgroundColor={cardBgColor}>
+      <Flex
+        justifyContent="space-between"
+        py={8}
+        ml={14}
+        pl={10}
+        pr={{
+          base: 10,
+          lg: 0,
+        }}
+        borderLeft="8px"
+        borderColor={borderColor}
+        position="relative"
+        direction={{
+          base: "column-reverse",
+          lg: "row",
+        }}
+        _after={
+          isFirst && {
+            content: `""`,
+            position: "absolute",
+            left: "-12px",
+            zIndex: "100",
+            top: "0",
+            width: "18px",
+            height: "50%",
+            background: cardBgColor,
+          }
+        }
+      >
+        <VStack
+          alignItems="start"
+          maxWidth={{ base: "100%", lg: "40%" }}
+          spacing={{
+            base: 18,
+            lg: isFirst ? 32 : 24,
+          }}
+        >
+          <VStack alignItems="start" spacing={0}>
+            {challengeStatus && (
+              <Badge borderRadius="xl" colorScheme={colorScheme} textTransform="none" py={0.5} px={2.5}>
+                {label}
+              </Badge>
             )}
-          </ButtonGroup>
-        ) : (
-          <ButtonGroup>
-            <Button
-              as={RouteLink}
-              to={!isChallengeLocked && `/challenge/${challengeId}`}
-              isDisabled={isChallengeLocked}
-              variant={isChallengeLocked ? "outline" : "solid"}
-              isFullWidth
-            >
-              {!isChallengeLocked ? (
-                <>
-                  {" "}
-                  <span role="img" aria-label="castle icon">
-                    ⚔️
-                  </span>
-                  <chakra.span ml={1}>Quest</chakra.span>
-                </>
-              ) : (
-                <>
-                  <span role="img" aria-label="lock icon">
-                    🔒
-                  </span>
-                  <chakra.span ml={1}>Locked</chakra.span>
-                </>
-              )}
-            </Button>
-            {!builderHasCompletedDependenciesChallenges && (
-              <Tooltip label={lockReasonToolTip}>
-                <IconButton icon={<QuestionOutlineIcon />} />
-              </Tooltip>
+
+            <Text color={primaryFontColor} fontSize="xl">
+              Challenge #{challenge.id}
+            </Text>
+            <Text fontSize="3xl" color={primaryFontColor} mt={0}>
+              {/* {challenge.label} */} Simple NFT Example
```

Comment:
I think you forgot to add this back.

Reply:
Yeah, Actually it was looking ugly while recording the video 😂, sorry for it
![Screenshot 2022-11-21 at 5 55 50 PM](https://user-images.githubusercontent.com/80153681/203059435-ec1fd285-b267-4b6b-a25a-10245b69b4e3.jpg)

There seems like there was a challenge number in the challenge title itself for first 3 challenges, removed the challenge number from those in https://github.com/BuidlGuidl/SpeedRunEthereum/pull/177/commits/7de6658b8f80a5ab2a4b6876b7c8771838790f90

Also in challenge data ( [challenges.js](https://github.com/BuidlGuidl/SpeedRunEthereum/blob/master/packages/react-app/src/data/challenges.js)  ), should I update the id's in order ? so that we use them efficiently to show challenge number?

Reply:
Maybe we need to tweak the portfolio page now, to show the "Challenge #number" on the challenges.

Now:
![image](https://user-images.githubusercontent.com/2486142/203139367-46694d95-5805-42bd-a79d-bcec5a4b46a0.png)

Live:
![image](https://user-images.githubusercontent.com/2486142/203139711-53dfae49-f943-48c1-a7dd-68e1293408b6.png)

No need to be in the same style! Maybe can something below (as in the new homepage). Up to you.

> Also in challenge data ( [challenges.js](https://github.com/BuidlGuidl/SpeedRunEthereum/blob/master/packages/react-app/src/data/challenges.js) ), should I update the id's in order ? so that we use them efficiently to show challenge number?

Might be a bit tricky to change that file, since it's used for a bunch of stuff (like the autograder). Let's leave it for now.

Reply:
Hii, I reverted the changes in `challenges.js` i.e added back the `🚩 Challenge 0: ` to the labels where they were used, and instead of changing other files like portfolio, I parsed the title in `ChallengeExpandedCard.jsx` itself...
```js
{challenge.label.split(": ")[1] ? challenge.label.split(": ")[1] : challenge.label}
```
Its a bit hacky 😅, please let me know what you think, like since this data was used in other files  too that's why I parsed it in `ChallengeExpandedCard.jsx` so that it won't affect other files

Reply:
I approve this hack :D We won't tell Uncle Bob.